### DISCUSSION TITLE
Questions about about the stock FW and AIO FW

### DISCUSSION THREAD

**Discussion by petflc** [Mr-MIBonk/M.I.B._More-Incredible-Bash#323] [Q&A] [answered]
Hi community,
I have a 2016 Audi Q7 with FW MHI2_ER_AUG22_S2140 MU0358, which doesn't have the smart phone interface. I'm trying to get this function abled if is possible, So base on my current version, can I just install the Audi AIO FW MHI2_ER_AUGX_K3663_1_MU1425_AIO? Or I need to update the stock FW to another version first and then run mib patches to active carplay function? If it needs an update, which FW should I update to?
![IMG_1466](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/143505065/83153581-47aa-45f8-9fc1-95776154bb0c)
![IMG_1467](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/143505065/7e8078ab-b6fb-42e3-bb8e-1cc21f69a77b)

Thanks in advance.
Regards Frank

**Comment by shelipso** [comment]
I have the same version and same issue.

I installed MIB and got the green screen working. I can now see the Audi Smartphone option in the menu but when tried, FW could not be patched - as FW was not found on the SD. Do you think I/you need to upload a backup of current FW so a patched version is created? or should I install AIO version of answer FW and then patch it to that FW. Also now I have lost my ability to connect smartphone via usb (bluetooth still works) and menu seems to be responding a bit slower. 

So any help on how to get the FW patched is welcome.

**Reply by petflc** [reply]
Hi shelipso,
Thanks for your reply, so you didn’t update any other FW before installing M.I.B? and you got the smartphone interface after installing M.I.B?

**Reply by shelipso** [reply]
Hi, yes but it does not connect via bluetooth and I lost my iphone usb connectivity to mmi (can not even play music via usb, the option is not shown in the menu any more). So now contemplating FW upgrade, not sure if I should upgrade to the latest available version for MHI2 or the first one that would enable CarPlay.

**Reply by petflc** [reply]
Hi shelipso,
I think If the FW is not patched that means the smartphone interface is still not available, the function is actually not be activated. Have you tried cope all different version’s patches to that patch folder and run to patch? Or if you go to engineering mode to check and see if you have CarPlay/Android Auto FSC code 60800/60900 valid?

**Reply by shelipso** [reply]
Hi Frank,

Still no luck. I had copied all the patches.

I tried SW update. I started with MHI2_ER_AUG22_K3344_MU1438 (A4-A5-Q5-Q7) but tried 3346 too. I cannot get it installed using the FW Update. The zip files don't have autorun files in them. So had to run them manually. I also checked the User SWL option both in  production menu and MIB of Green Engineering Menu but here is the error I get when trying to upgrade the FW.

Not sure what is next but still have lost the usb data connections. Interesting that the added Audi Smart Phone which is non functional takes me to connection manager. In there I have the Audi Smart Phone but the connection methods are not working.

![image](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/129860502/79541f3f-c2e3-49bd-9f5c-66cbabc2c435)

**Reply by shelipso** [reply]
Just an update, I found that MHI2_ER_AUG22_K2161_MU1421 (A4-A5-Q5-Q7) has S2140 in its supported trains. So can try that. Problem is MIB patches, don't include K2161 so not sure if that helps.

Have you downloaded K3663_1_MU1425 ? Does include S2140 as supported train? or can it be installed with this support? Also do you know what is the difference between the ones having _AIO and others. Is it just the full compilation?

![image](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/129860502/df15b5b0-4eb4-49b5-ad3b-4b3e1d25f1c6)

**Reply by shelipso** [reply, marked as answer]
Right, it finally worked! I cannot believe I could have done this sooner.

Updated FW to MHI2_ER_AUG22_K2161_MU1421 (A4-A5-Q5-Q7). Just copy the folder to the SD Card (follow instructions) and add Gracenotes2. No need for any other change. Upgrade FW using RED MENU. Then, create another SD Card with M.I.B and make sure the 2161 patch is in the patch folder. Once it is installed, patch it from AIO Patch menu of the MIB from GEM.
 
![image](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/129860502/36e90595-ffe1-41ae-a331-ecf55883dc46)

**Reply by shelipso** [reply]
Thanks to all who have developed the patches and organised the site. Shame Audi would have not done that, even if you were happy to pay.

**Reply by petflc** [reply]
Hi Shelipso,
Congratulations first, thanks for your keep trying and sharing your final solution with community. I’ll try your way once I got a chance.

**Reply by AudiQ750** [reply]
Hi There I managed to get my system updated finally after following the same steps now I have AA and Carplay. The only issue I got Is no sound :(. I was sure the Bose system was not selected. Everything works as it should except for no sound. Is there any way to fix this without a trip to the dealer? The fault code in the system for 47 Sound is B201200. Tried clearing but cant. Any ideas?

**Reply by AudiQ750** [reply]
Ok so I have found a solution to the no sound after the update. As I flashed the bose system when doing MMI update it upload incorrect files to the MMI therefore no sound output. I contacted a tech from the UK and he asked for my Bose Amp model number (Located under Amp) and sent me the correct file and now I have sound :). Three weeks with no sound and now audio is working. Msg this guy on FB https://www.facebook.com/zubscoding and he will sort the sound file for your system. Ps I went to the Steelership and they told me I need a new MMI system @13K or amp @4.5K. They had no idea.

**Reply by googleg** [reply]
Wow this is scary, I thought the AIO was specifically configured NOT to update the Bose system.

**Reply by Kalipso341** [reply]
Hi, Thanks for the info. I was able to update from S2140 to K2161 (I'm not sure about the Gracenote2). When I try the MIB install, the REM doesn't recognise the SD Card! Can you help, please?

**Reply by N9130** [reply]
> Thanks to all who have developed the patches and organised the site. Shame Audi would have not done that, even if you were happy to pay.

Where did you find the patch, is it a new download?

**Reply by CQH** [reply]
> Ok so I have found a solution to the no sound after the update. As I flashed the bose system when doing MMI update it upload incorrect files to the MMI therefore no sound output. I contacted a tech from the UK and he asked for my Bose Amp model number (Located under Amp) and sent me the correct file and now I have sound :). Three weeks with no sound and now audio is working. Msg this guy on FB https://www.facebook.com/zubscoding and he will sort the sound file for your system. Ps I went to the Steelership and they told me I need a new MMI system @13k or amp @4.5K. They had no idea.

Whelp. I find myself in the same situation with my Porsche.

I've reached out Zubs Coding, but I haven't heard back yet. 

Can you tell me what the procedure was for your fix?

**Reply by obasha** [reply]
Hi, Just chiming in here in case anyone else lost audio on there Bose system. The fix was actually pretty straight forward, No need to pay the opportunists, goes against the grain of this project. 
Step 1: Download the Bose 430 file from: https://mibsolution.one/#/1/9/MHI2%20-%20HARMAN/Firmware/Audi 
If the link doesnt work, goto mibsolution.one login as per normal with guest account. Navigate to MQB Solutions-> MHI2 - HARMAN-> Firmware-> Audi. In this folder you will find the Bose files. I used the 430 as the upgrade I used was 440(The one that bricked myne.)
Download the zip file, 
Step 2: 7z extract downloaded file.
Step 3: Copy contents of folder to a Fat32 formatted sd card(Normal way of formatting for MIB etc). 
Step 4: Insert card in to SD1 slot in car. 
Step 5: Switch on ignition.
Step 6: Enter Red menu
Step 7: Once in red menu you will see only the Bose system available to update. Go ahead and scroll down and start the update. Unit will reboot after its done, takes a good few mins and maybe two or three reboots.
Step 8: Done. Once the unit reboots it will be working again.
Disclaimer: The above worked for me. If it doesnt work for you my apologies, this is a best effort on my part, I do not do this as a hobby, just sharing what worked for me. I avoided needing to pay 150 pounds for someone to send me the files and potentially the steps above, Hopefully I'm assisting others.

**Comment by N9130** [comment]
Updating for 2024 as Shelipso does really give a break down of what steps to complete, so not helpful for others!!. 

help you to activate Apple CarPlay / Android Auto with Mib2 units.

Supported units with firmware starting from:

MHI2
MHI2Q

not supported: some of the below will require retrofits or adapters 

HN+ / HN+R / Hnav / Bnav 
MH2S / MHIG / MENT / MSTD
MMI 2G High
RMC

Supported firmware versions
Only those firmware are supported by script. If you don’t have that firmware, you need to update first.

MHI2Q_CN_AUG22_K1004_MU1003
MHI2Q_CN_AUG22_P0915_MU0915
MHI2Q_CN_AUG22_P1002_MU1002
MHI2Q_ER_AUG22_P3638_MU0916
MHI2Q_ER_AUG22_P3638_MU0917
MHI2Q_ER_AUG22_P4170_MU1117
MHI2Q_ER_AUG22_P4212_MU1177
MHI2Q_ER_AUG22_P5092_MU1329
MHI2Q_ER_AUG22_P5152_MU1329
MHI2Q_JP_AUG22_P1017_MU1017
MHI2Q_KR_AUG22_P1018_MU1018
MHI2Q_US_AUG22_P4171_MU1118
MHI2Q_US_AUG22_P4213_MU1178
MHI2Q_US_AUG22_P4246_MU1320
MHI2Q_US_AUG22_P5087_MU1316
MHI2_CN_AU276_P1214_MU1214
MHI2_CN_AU276_P1215_MU1215
MHI2_CN_AU57x_K1006_MU1005
MHI2_CN_AU57x_P0623_MU0623
MHI2_CN_AU57x_S0037_1_MU0037
MHI2_CN_AUG22_K0730_MU0729
MHI2_CN_AUG22_P0618_MU0618
MHI2_CN_AUG22_S0038_MU0038
MHI2_CN_AUG22_S0039_MU0039
MHI2_ER_AU276_P5088_MU1325
MHI2_ER_AU276_P5502_MU1460
MHI2_ER_AU276_P5506_MU1466
MHI2_ER_AU276_P5509_MU1474
MHI2_ER_AU276_P5510_MU1477
MHI2_ER_AU276_P5515_MU1488
MHI2_ER_AU37x_P3620_MU0875
MHI2_ER_AU37x_P4159_MU1106
MHI2_ER_AU37x_P4203_MU1168
MHI2_ER_AU37x_P5089_MU1326
MHI2_ER_AU37x_P5153_MU1326
MHI2_ER_AU43x_P5098_MU1339
MHI2_ER_AU43x_P5151_MU1339
MHI2_ER_AU43x_P5404_MU1479
MHI2_ER_AU43x_P5406_MU1489
MHI2_ER_AU57x_K2589_MU1429
MHI2_ER_AU57x_K3663_MU1425
MHI2_ER_AU57x_P3236_MU0670
MHI2_ER_AU57x_P3634_MU0910
MHI2_ER_AU57x_S2519_MU0389
MHI2_ER_AU62x_P5099_MU1340
MHI2_ER_AUG22_K2161_MU1421
MHI2_ER_AUG22_K3344_MU1438
MHI2_ER_AUG22_K3346_MU1438
MHI2_ER_AUG22_P3241_MU0678
MHI2_ER_AUG22_S2148_MU0442
MHI2_JP_AU43x_P0902_MU0902
MHI2_JP_AU62x_P0701_MU0616
MHI2_JP_AUG22_P0713_MU0713
MHI2_KR_AU37x_P0901_MU0818
MHI2_KR_AU57x_P0901_MU0615
MHI2_KR_AU57x_S0035_MU0035
MHI2_KR_AUG24_S0008_MU0008
MHI2_TW_AU37x_P0901_MU0816
MHI2_US_AU276_P5083_MU1312
MHI2_US_AU37x_K5512_1_MU1484
MHI2_US_AU37x_P5107_MU1363
MHI2_US_AU43x_K1083_MU1412
MHI2_US_AU43x_P5124_MU1389
MHI2_US_AU43x_P5180_MU1462
MHI2_US_AU57x_K3324_1_MU0925
MHI2_US_AU57x_K3341_MU1426
MHI2_US_AU62x_K2588_MU1407
MHI2_US_AU62x_P5125_MU1390
MHI2_US_AU62x_P5181_MU1463
MHI2_US_AUG22_K3345_MU1439
MHI2_US_AUG22_K3347_MU1439
Europe / Australia & New Zealand / Middle East / RoW / Singapore / South Africa
[MHI2_ER_AUG22_K3344]
[MHI2Q_ER_AUG22_P5092]
[MHI2_ER_AU57x_K3663]
USA / Canada
[MHI2_US_AUG22_K3347
[MHI2Q_US_AUG22_P5087]


1, Check the MMI unit your car has, by entering red menu and viewing system> if you are a supported unit move onto the below steps
2, get yourself an SD card 16GB is more then enough and an SD reader if you don't have one. - card with need to be formatted to fat32
3, go to mib-helper and type in the SW version you are running , this will tell you the newest version you can go to
4, download the FW from mibsolution.com username:guest password:guest
5, extract the contents of the downloaded FW to the SD card using 7 zip
6, make sure all the folders are at the top level of the card and not within the FW folder name
7, turn on your car and place the SD card into SD1
8, enter RED menu on your car
9, select update and allow the update to complete and finish (takes roughly 20-40 minutes) system will reboot a few times

10, re format the SD card you have with fat32 again
11, download MIB; download can be found here: https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash
12, extract the contents of the mib download
13, make sure all the folders are at the top level of the card and not within the FW folder name
14, turn on your car and place the SD card into SD1
15, enter RED menu on your car
16, select update and allow the update to complete and finish (takes roughly 20-40 minutes) system will reboot a few times
17. GEM menu will now be unlocked - you are able to activate CP and AA from here - (but doesn't mean its enabled)

18, to enable CP/AA will need to download a patch for the FW you downloaded
19, download the FW patch from mibsolution.com username:guest password:guest
20, found under MIB>20230926_M.I.B_Patches.7z
21, extract this folder anywhere this will contain loads of FW patches 
22, find you patch number same as FW you downloaded. and extract the files within the folder to the SD card under patches /patches/
23, turn on your car and place the SD card into SD1
24, enter GEM menu on your car
25, select update and allow the update to complete and finish (takes roughly 20-40 minutes) system will reboot
26, congrats you know have APPLE CAR PLAY AND ANDRIOD AUTO!!!!!!

**Reply by Umid9405** [reply]
Hi, I have Audi Q7 4M, I did everything as per your instruction, Audi smartphone interface activated. But when im trying to connect USB (I have 2 ports and 1 Aux, 1 port for iPod) no responses, i tried both AA and Carplay not working only message - no devise is connected which support the Audi smartphone interface etc. USB originals for Android and for Iphone working in other cars.

**Reply by N9130** [reply]
Hey, have you completed the patch part? Even though you can activate Carplay/AA. It doesn't work till you have patched your system. 

18, to enable CP/AA will need to download a patch for the FW you downloaded
19, download the FW patch from mibsolution.com username:guest password:guest
20, found under MIB>20230926_M.I.B_Patches.7z
21, extract this folder anywhere this will contain loads of FW patches
22, find you patch number same as FW you downloaded. and extract the files within the folder to the SD card under patches /patches/
23, turn on your car and place the SD card into SD1
24, enter GEM menu on your car
25, select update and allow the update to complete and finish (takes roughly 20-40 minutes) system will reboot

**Reply by Umid9405** [reply]
![1726636588165](https://github.com/user-attachments/assets/8b936e55-890e-4dad-9d3b-a195b026ad7d)
![1726636588168](https://github.com/user-attachments/assets/6559792b-ef5e-488c-a56d-4fd61eedb31b)
During the patch: Header size on unit - 2479E900 - does not match file size 2C79E900 etc.

**Reply by Umid9405** [reply]
The issue was that the built-in file compression tool of Windows was not extracting the necessary files from the patch. Using Winrar to extract the patch files solved this.

**Reply by N9130** [reply]
Sorry for the late reply. Work been crazy. Glad it's sorted and hopefully it'd all working as expected for you.  Yeah built in windows extractors csn do that sometime. I have 7zip/with zip installed for my extracting and uses as default. So didn't see anything like that on mine.

**Reply by brankezan** [reply]
Until today I had no option for CA even though my car is 2018 q5 and should already have it by default. I also did all steps above, but now I got Audi Smartphone option (which i didnt have before). it’s not blur, but I cant click on it to connect my phone. Anyone had similar situation? Thanks in advance 🙌🏻

**Reply by dkomdus** [reply]
So many thanks!!
I ve updated from MHI2_ER_AUG22_S2140 MU0358 to MHI2_ER_AUG22_K2161_MU1421 but had to install first the mib to get the gem menu and enable option to choose an update without the bose files in the red menu.
Then i used the correct patch.
It works fully, thanks a lot guys, i was searching this for 5 years and thought it wasn t possible !

**Reply by puni746** [reply]
@dkomdus so did you download the MIB and install it, so you could just activate the GEM? with still that same MHI2_ER_AUG22_S2140 MU0358 then updated after you had the GEM to K2161 MU1421? I need to then also do the same so I dont get the no audio from the Bose?

**Reply by puni746** [reply]
@N9130 I also have MHI2_ER_AUG22_S2140 MU0358. Did you just update to the MHI2_ER_AUG22_K2161_MU1421 version then installed the MIB and follow from step 10 onwards? Why I am asking is: Will this steps give me the Bose No sound problem?

**Comment by N9130** [comment]
![Screenshot_20240920_131444_Chrome](https://github.com/user-attachments/assets/787e5305-e321-4b04-bde7-9af94e95d258)


Not my picture, but do you have this within your connection manager? My q7 and a3. Have this. I need to connect my phone via a USB cable and complete activate for CP/AA?

**Reply by shelipso** [reply]
Hi All, just catching up with this thread. I have still the Q7 and Carplay is still active via the Audi smart connection. That menu item turns automatically into Apple Car Play once connected. So once it works, it recognizes it as soon as the cable is connected. I did not have some of the issues noted by N1390 and others. Mine was a 2015 build so one of the earlier builts. Has there been a change to hardware that have made it difficult to upgrade? Also the other experience was that it did not work with some cheap charge only cables. I can have a look at the structure of the SD card (I think I still have it) and list what I had there if it helps but given almost 2 years passed, cannot remember all details. Sorry this may not be very helpful now!

**Reply by Meyertrix** [reply]
Hi Shelipso,
I’m having trouble with the first step of updating the firmware to the version supported by MIB. Could you let me know where you downloaded the firmware from? If it’s not available, would you mind sharing yours? I have the same version but need to patch it to enable CarPlay.

Edit
Nevermind, I found it under the Mibsolution.one website /MQB_Solution/MHI2 - HARMAN/Firmware/Audi

**Reply by Meyertrix** [reply]
Success! I updated the firmware and applied the MIB patch. Everything is working smoothly with no issues.

Thank you @shelipso you guided me in the right direction in getting this working.

**Comment by brankezan** [comment]
Unfortunatelly, its not the case since when I click on Audi smartphone, there is no dropdown menu. I got 15Pro with latest iOs, but can't find what potentially could be an issue. Maybe with factory settings and installing everything from scratch would solve it?

**Reply by N9130** [reply]
Hi Brankezan,

Try with another USB cable. I have a Samsung, but uses a onerous fast charger cable. 

Does you phone not give this message? Like mine. Pictures attached. 
![20240926_160637](https://github.com/user-attachments/assets/285f8bac-bd63-40a6-87d9-852010da5d23)
![20240926_160620](https://github.com/user-attachments/assets/7dbeeda9-3332-4841-818f-8d372ef2a81b)
![Screenshot_20240926_160555_One UI Home](https://github.com/user-attachments/assets/a6971d74-b510-4f38-9225-a30f943a80a0)

**Reply by brankezan** [reply]
Tried both apple and samsung cables, nothing. Wifi adapter also nothing. I have only 1 usb socket and 1 aux socket in my car. Is that maybe an issue? Here it's how it looks like 
![image_50390017](https://github.com/user-attachments/assets/56515e83-f90b-4cce-a420-404b9d2657ab)
![image_50416129](https://github.com/user-attachments/assets/778a71b6-cf77-4086-9c93-921a007e9bcb)
![image_50396673](https://github.com/user-attachments/assets/ab858112-a5d6-40bc-abd0-24e7bc8f37a4)

**Reply by N9130** [reply]
Hi Brankezan, interesting. Just asking but have you completed both the patch install and activation using MIN "Green Menu"? 

Also with the MIB "green Menu" there is a section that's shows with ports have been activated. In the q7 you have two in the arm rest, plus other on the back, not sure on a q5. I will get MIB install back on a sd card and check. But try looking at that. Or the only other things I can suggest is maybe try activation and patch again

**Reply by Meyertrix** [reply]
Make sure to update the firmware before patching the system. The firmware version must match the version in the patches folder for the process to work correctly. You’ll need two separate memory cards: one for the M.I.B._More-Incredible-Bash-3.6.0 and another for the specific firmware compatible with your car. I hope this helps, as I initially struggled with this, thinking the M.I.B. patch would update the firmware, but that’s not the case.
Firmware is available at - https://mibsolution.one/#/1/9/MHI2%20-%20HARMAN/Firmware/Audi
Make sure to follow the steps in creating the card and updating the system from the Red Engineering menu. FW update tutorial here - https://www.youtube.com/watch?v=xDAw4sj-9tI&feature=youtu.be

**Comment by Alex320dd** [comment]
Hallo,
Ich suche MIB>20230926_M.I.B_Patches.7z
Zum runter laden.
Die offizielle Seite ist nicht mehr erreichbar.
Danke

**Comment by AudiQ750** [comment]
He gave me the correct sound files which I flashed into the system. Took 5
mins.

**Reply by CQH** [reply]
FYI: The dealership did it for me. I simply asked them to reflash my head unit back to the original firmware. They did it and only charged me for 1 hr of labor.

**Comment by liviua76-svg** [comment]
Hi,
If some can help, which M.I.B folder i have to use after MHI2_ER_AUG22_k2161 updated ?

**Comment by Drenica129** [comment]
Hey, what exactly do you need? CarPlay activation or something else?

liviua76-svg ***@***.***> schrieb am Di., 17. März 2026,
17:55:

**Reply by liviua76-svg** [reply]
Carplay activation.
I have updated the Fw with mhi2_er_aug22_k2161 and mib v3.6 with same patches in patch folder as the fw but still doesn’t work

**Comment by Drenica129** [comment]
Write to me on WhatsApp on this number 01729289245. I'll help you.

liviua76-svg ***@***.***> schrieb am So., 22. März 2026,
16:46:

**Comment by dacafeplayboy-cmd** [comment]
I could use a little help as well. When was the firmware ledger last updated? I recently got an official update to maps and firmware through Audi. (February 2026). Based on MIB solutions I have the last version.  I dont see mine listed (MHI2_US_AUG11_SO533_0143). Iv also seen google role out support for HUD navigation. Is that something this hack can link as well?

**Comment by Drenica129** [comment]
Hello, write me a WhatsApp.+491729289245

puni746 ***@***.***> schrieb am So., 6. Sept. 2026, 17:59:

**Comment by dkomdus** [comment]
Yeah that’s it.

 

Without the MIB I didn’t have the option to disable audio files in the update.

 

So :

- 1st install of the MIB to get the GEM menu

 

-Update withtout bose files,

 

-2nd install of the MIB after the update,

 

-Patch.

 

Bose works fully since the instal.

 

Cheers. 

 

 

I ve updated from MHI2_ER_AUG22_S2140 MU0358 to MHI2_ER_AUG22_K2161_MU1421 but had to install first the mib to get the gem menu and enable option to choose an update without the bose files in the red menu.
Then i used the correct patch.

 

De : puni746 ***@***.***> 
Envoyé : dimanche 6 septembre 2026 18:00
À : Mr-MIBonk/M.I.B._More-Incredible-Bash ***@***.***>
Cc : dkomdus ***@***.***>; Mention ***@***.***>
Objet : Re: [Mr-MIBonk/M.I.B._More-Incredible-Bash] Questions about about the stock FW and AIO FW (Discussion #323)

 

@dkomdus <https://github.com/dkomdus>  so did you download the MIB and install it, so you could just activate the GEM? with still that same MHI2_ER_AUG22_S2140 MU0358 then updated after you had the GEM to K2161 MU1421? I need to then also do the same so I dont get the no audio from the Bose?

—
Reply to this email directly, view it on GitHub <https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/discussions/323?email_source=notifications&email_token=BRAGYEOX2YHU54JKPOQCWJ35NWCXTA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCOBTGE4DINRSUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSWGM33PORSXEX3DNRUWG2Y#discussioncomment-18318462> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/BRAGYEKL6E5VTM252IEOEGL5NWCXTAVCNFSNUABHKJSXA33TNF2G64TZHM2DIOJWGE2DCNRVHNCGS43DOVZXG2LPNY5TKNJXGA3TQOFBOYBA> .
Triage notifications, keep track of coding agent tasks and review pull requests on the go with GitHub Mobile for iOS <https://github.com/notifications/mobile/ios/BRAGYEM6FMEIZVJTDOVK4HL5NWCXTA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCOBTGE4DINRSUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSVGM33PORSXEX3JN5ZQ>  and Android <https://github.com/notifications/mobile/android/BRAGYEIKNEOGCQRZ375BL535NWCXTA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCOBTGE4DINRSUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSXGM33PORSXEX3BNZSHE33JMQ> . Download it today!

**Reply by puni746** [reply]
@dkomdus So if I understand you correctly : Install just MIB with no patch or FW update. Then that just activates GEM? 

Then I download Newest Firmware to SD Card, then in Red Menu I run update, then I get the new FW Patch, add it to the Patch folder and run it inside GEM with the MIB Launcher to update and activate everything? 

I see newest software is MHI2_ER_AUG22_P3346 or should I use MHI2_ER_AUG22_k2161

**Comment by dkomdus** [comment]
NO,

 

With these steps you will avoid bose non sound problem.

 

The no sound bose comes from the update, you must updat without the sound package.

 

De : puni746 ***@***.***> 
Envoyé : dimanche 6 septembre 2026 18:12
À : Mr-MIBonk/M.I.B._More-Incredible-Bash ***@***.***>
Cc : dkomdus ***@***.***>; Mention ***@***.***>
Objet : Re: [Mr-MIBonk/M.I.B._More-Incredible-Bash] Questions about about the stock FW and AIO FW (Discussion #323)

 

@N9130 <https://github.com/N9130>  I also have MHI2_ER_AUG22_S2140 MU0358. Did you just update to the MHI2_ER_AUG22_K2161_MU1421 version then installed the MIB and follow from step 10 onwards? Why I am asking is: Will this steps give me the Bose No sound problem?

—
Reply to this email directly, view it on GitHub <https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/discussions/323?email_source=notifications&email_token=BRAGYEIGFR4SD7SVILOLMOT5NWEFJA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCOBTGE4DKOJUUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSWGM33PORSXEX3DNRUWG2Y#discussioncomment-18318594> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/BRAGYEN4EVV36HVSQIF7IT35NWEFJAVCNFSNUABHKJSXA33TNF2G64TZHM2DIOJWGE2DCNRVHNCGS43DOVZXG2LPNY5TKNJXGA3TQOFBOYBA> .
Triage notifications, keep track of coding agent tasks and review pull requests on the go with GitHub Mobile for iOS <https://github.com/notifications/mobile/ios/BRAGYELEW54AXGAMIUDC3G35NWEFJA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCOBTGE4DKOJUUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSVGM33PORSXEX3JN5ZQ>  and Android <https://github.com/notifications/mobile/android/BRAGYELYUTJ2H5LYQNIMSF35NWEFJA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCOBTGE4DKOJUUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSXGM33PORSXEX3BNZSHE33JMQ> . Download it today!
